src/main/java/ch/njol/skript/effects/EffContinue.java

```diff
@@ -48,8 +47,19 @@
 		Skript.registerEffect(EffContinue.class, "continue [loop]");
 	}
 
-	@SuppressWarnings("null")
-	private Loop loop;
+	@SuppressWarnings("NotNullFieldNotInitialized")
+	private LoopSection loop;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		List<LoopSection> loops = getParser().getCurrentLoops();
```

Comment:
While this may be stuff for another PR, I feel like adding support for the while section is very easy (unless there needs to be an explicit discussion, which I would understand).

Reply:
I also feel like this would fit better in a different PR, you should open an issue for it